geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/protobufs/TupleProtoBuf.scala

```diff
@@ -50,6 +74,17 @@ trait TupleProtoBuf {
         multibandTileProtoBufCodec.decode(message.tiles.get))
   }
 
+  implicit def tupleSpatialKeyImageProtoBufCodec =
+    new ProtoBufCodec[(SpatialKey, Array[Byte]), ProtoTuple] {
```

Comment:
That would make things a bit easier. Is this something that should be implemented in this PR? Or just something to consider for later?